.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.sln]
+indent_style = tab
+
+[*.{json,csproj,props,targets}]
+indent_size = 2
```

Comment:
I'd personally just keep this at four, but I'm not incredibly pressed on it.

Reply:
@codymullins Specifically for JSON or in general? I'm okay with 2 for JSON.